apps/frontend-manage/src/components/questions/tags/SuspendedTags.tsx

```diff
@@ -27,11 +31,33 @@ interface Props {
 function SuspendedTags({ showUntagged, activeTags, handleTagClick }: Props) {
   const t = useTranslations()
 
+  const [searchInput, setSearchInput] = useState('')
+
   const { data, error } = useSuspenseQuery(GetUserTagsDocument)
   const [updateTagOrdering] = useMutation(UpdateTagOrderingDocument, {
     refetchQueries: [{ query: GetUserTagsDocument }],
   })
 
+  const { handleSearch } = useSortingAndFiltering()
+
+  const filteredTags = useMemo(() => {
+    if (data?.userTags) {
+      const search = new JsSearch.Search('id')
+      search.searchIndex = new JsSearch.TfIdfSearchIndex('id')
+      search.indexStrategy = new JsSearch.AllSubstringsIndexStrategy()
+      search.addIndex('name')
+      search.addDocuments(data.userTags)
+
+      if (searchInput) {
+        return search.search(searchInput) as Tag[]
```

Comment:
Typecasting should only be required here - the two `as Tag[]`s can be removed to make sure that the standard tag type is still correct

Reply:
unfortunately, proper typing for this library does not seem to be available otherwise...